macros/multiple_databases/to_varchar.sql

```diff
@@ -3,7 +3,11 @@
 {%- if target.type == 'snowflake' -%}
     to_varchar({{ field }})
 {%- elif target.type == 'sqlserver' -%}
+    case when charindex(' ', {{ field }})>0
+    then convert(nvarchar(2000), {{ field }})
+    else
     nullif(convert(nvarchar(2000), {{ field }}),'')
+    end
```

Comment:
Please use indenting on the lines and spaces between the `>`  to format this case statement.